src/main/java/com/exercise/basic/Bai8.java

```diff
@@ -0,0 +1,14 @@
+package com.exercise.basic;
+
+public class Bai8 {
+
+    public int sumElements(final int arr[]) {
+        int sumResult = 0;
+
+        for (int i = 0; i < arr.length; i++) {
+            sumResult += Math.pow(arr[i], 2);
```

Comment:
Mình không nên lạm dụng hàm này vì bản chất của hàm này là tính trên số thực và nó là kết quả gần đúng

![image](https://user-images.githubusercontent.com/10450325/216647633-95d96da1-98a7-4f45-b244-d432dbb82ea7.png)


```suggestion
            sumResult += arr[i] * arr[i];
```

Reply:
Done